style.css

```
@@ -2232,3 +2232,204 @@ nav, nav.open {
+    --cv-black:  #0F0F0F;
+    --cv-brown:  #362C28;
+    --cv-gold:   #C29E5D;
+    --cv-teal:   #219EBC;
+    --cv-warm:   #D8CABF;
+    --cv-clay:   #DB3A34;
+    --cv-signal: #FFDD57;
+}
+
+/* ── Typography overrides ──────────────────────────────── */
+.cv-page .primary-font-title {
+    font-family: 'Space Grotesk', 'Syne', sans-serif !important;
```

Stylelint (17.5.0)

[error] Unexpected quotes around "Syne" (font-family-name-quotes)

(font-family-name-quotes)